### PR TITLE
Palindrome checker Project 

### PR DESCRIPTION
**Pull request by Sharifarezaie**
My Palindrome checker project is including index.html, styles.css, script.js also linters config files and readme.

### CODE REVIEW

**Comment by Sharifarezaie** [comment]
Hi dear ,i hop you are well
please check my project!

**Comment by Sharifarezaie** [comment]
@fatiabdulahi
Hi dear i hope you are will
please check my project

**Comment by fatiabdulahi** [comment]
Hello dear sharifa .I review your project. I think that's good .well done
provide

**Comment by Sharifarezaie** [comment]
@ahzamir
could you please review my project sir?

**Review by ahzamir — APPROVED** [review]
### Hi @Sharifarezaie 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Although I approved your project still your README.md file has some issues, please have a look on it solve them and then merge your branch.

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF (2)

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Palindrome Checker project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your repo has a professional README.md file
✔️Your code is clean
✔️Your design is impressive and responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+open the project on your IDE like vscode :
+
+cd>palindrom-checker-project> code .
+Features ⭐
+Responsive webpage
+sharifaRezaie👩‍💻
+LinkedIn: [(https://www.linkedin.com/in/sharifa-rezaie-646636309/)]
+Email: [(sharifashaida82@gamil.com)]
+Contributing 🤝
+For contribution you can create a pull request and mention me there.Thank you.
```
Comment (ahzamir) [inline README.md]: - Your readme file has some info about your project but it's not ordered correct and does not have headings so add some heading using this sign(###)

**Comment by Sharifarezaie** [comment]
OK thank you sir

**Pull request merged**